symphony-bdk-core/src/main/java/com/symphony/bdk/core/util/exception/PresentationMLParserException.java

```diff
@@ -0,0 +1,14 @@
+package com.symphony.bdk.core.util.exception;
+
+import org.apiguardian.api.API;
+
+/**
+ * Exception thrown when the {@link com.symphony.bdk.core.util.PresentationMLParser} failed to parse a PresentationML string.
+ */
+@API(status = API.Status.STABLE)
+public class PresentationMLParserException extends Exception {
```

Comment:
That would be nice to add a `private final String presentationML;` field as well. A developer catching this exception could easily see why it failed.